### PR TITLE
Added testcase for SAI

### PR DESCRIPTION
**Pull request by mpeterss**

### CODE REVIEW

**Review by vihangpatil — APPROVED** [review]
That's easy to review. :)

File: ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt [hunk 1]
```diff
@@ -30,6 +30,18 @@ class UserLocationTest {
         assertEquals("16", userLocation?.mnc)
     }
 
+    @Test
+    fun userLocationParserMalaysiaSai() {
+
+        val locationHexString = "0105F26182BF804E"
```
Comment (la3lma) [inline ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt:36]: What does his magic hex string mean?

File: ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt [hunk 2]
```diff
@@ -30,6 +30,18 @@ class UserLocationTest {
...
+
+        val userLocation = UserLocationParser.getParsedUserLocation(locationHexString.hexStringToByteArray())
+
+        assertEquals("1", userLocation?.geographicLocationType)
+        assertEquals("502", userLocation?.mcc)
```
Comment (la3lma) [inline ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt:41]: MCC= malayisia, MNC = digi?   If true, then  be stated in comment

**Review by la3lma — COMMENTED** [review]
Suggested comments for better readability

File: ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt (hunk 1 above)
Comment (mpeterss) [reply ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt:36]: This is the Hex value of the blob we get in the 3GPP-USER-LOCATION-INFO AVP.

File: ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt (hunk 2 above)
Comment (mpeterss) [reply ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt:41]: It is in the name of the test. I can add Digi as well

**Pull request merged**